### DISCUSSION TITLE
About our repo:dev_zero_to_hero

### DISCUSSION THREAD

**Discussion by WinstonCHEN1** [EZEORG/dev_zero_to_hero#8] [Ideas]
好像教程很久没有更新了，大家是否有什么需要且泛用的内容，需要一份文档记录或教学？

**Comment by eze-root** [comment]
推荐几个深度的教程 

1. Git的日常使用
2. Python的语法糖 或者高级主题 （异步）
3. Github Action的讲解
4. RAG技术，这个有博士师兄在写。

**Reply by eze-root** [reply]
To see [this link](https://github.com/majihao/My_dev/tree/learn) for more information about RAG.

**Comment by eze-root** [comment]
另外，如果能找个机会打通markdown和latex，就能直接生成pdf文档了。改一下就可以用来写书了。

感觉还是得多写才能提升自我。
